### PR TITLE
Knowing when to ask for help

### PR DESCRIPTION
**Pull request by heathd**
first stab at this

also relates to #38

### CODE REVIEW

**Comment by quis** [comment]
Unrelated to this specific PR, but do you know why PRs from this repo are showing up on our Fourth Wall screen?

**Comment by heathd** [comment]
not sure @quis I guess it depends on your fourth-wall config

File: career-path/competencies/knowing-when-to-ask-for-help.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Knowing when and how to ask for help. Developing a [growth mindset](https://www.ted.com/talks/carol_dweck_the_power_of_believing_that_you_can_improve) that recognises that we learn from failure and feedback.
+
+### How do I learn/get good at it?
+
+Practice! Ask questions! But also reflect upon how effective your question-asking has been. Perhaps you need to do some research first? Maybe it's better to put your question in writing and then talk it through? Do you need to reproduce or isolate a problem so that you can show it to someone else?
+
+### Junior level
+
+You'll usually be given [well defined tasks](/career-path/competencies/leading-on-stories.md#junior-level) to work on, with support from a more experienced developer. Discuss the work with your colleague and try to understand what's involved.
```
Comment (Davidslv) [inline career-path/competencies/knowing-when-to-ask-for-help.md]: I feel that we need to cover what "well defined tasks" mean.
I can point to a number of tasks where the only thing written has been a title. I think we should create a document explaining what we mean with "well defined tasks" and who is responsible to write them

File: career-path/competencies/knowing-when-to-ask-for-help.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Asking for help can be hard, especially if you're new and perhaps experiencing [imposter syndrome](https://medium.com/learn-love-code/developers-how-to-overcome-imposter-syndrome-48edee803cf4).
+
+### What is it?
+
+Knowing when and how to ask for help. Developing a [growth mindset](https://www.ted.com/talks/carol_dweck_the_power_of_believing_that_you_can_improve) that recognises that we learn from failure and feedback.
+
+### How do I learn/get good at it?
+
+Practice! Ask questions! But also reflect upon how effective your question-asking has been. Perhaps you need to do some research first? Maybe it's better to put your question in writing and then talk it through? Do you need to reproduce or isolate a problem so that you can show it to someone else?
```
Comment (MatMoore) [inline career-path/competencies/knowing-when-to-ask-for-help.md]: @danielroseman has a slide deck about how to ask good questions, perhaps we could incorporate some of his advice?

I think we should link in the XY problem here: https://meta.stackexchange.com/questions/66377/what-is-the-xy-problem

**Pull request merged**